examples/get-view-from-dataset-demo-filter-argument-set.js

```diff
@@ -0,0 +1,131 @@
+const { Parser } = require('n3')
+const rdf = require('rdf-ext')
+const { ViewBuilder } = require('../lib/builders.js')
+const { Source } = require('../index.js')
+
+async function main () {
+  const source = new Source({
+    endpointUrl: 'https://ld.integ.stadt-zuerich.ch/query'
+  })
+
+  const { dataset, term } = getSampleData()
+  const { view } = ViewBuilder.fromDataset({ dataset, source, term })
```

Comment:
This file is a demo showing how argument sets are loaded from rdf to the view. It's not part of the lib, but an example.